source/com.google.android.datatransport/transport-runtime/Additions/Xamarin.Google.Android.DataTransport.Runtime.Dagger.Internal.ProviderOfLazy.cs

```diff
@@ -13,15 +13,15 @@ public sealed partial class ProviderOfLazy //: global::Java.Lang.Object, global:
     {
 
  		// Metadata.xml XPath method reference: path="/api/package[@name='com.google.android.datatransport.runtime.dagger.internal']/class[@name='ProviderOfLazy']/method[@name='get' and count(parameter)=0]"
-		[Register ("get", "()Lcom/google/android/datatransport/runtime/dagger/Lazy;", "")]
-		public unsafe global::Java.Lang.Object Get ()
-		{
-			const string __id = "get.()Lcom/google/android/datatransport/runtime/dagger/Lazy;";
-			try {
-				var __rm = _members.InstanceMethods.InvokeAbstractObjectMethod (__id, this, null);
-				return (Java.Lang.Object) global::Java.Lang.Object.GetObject<global::Xamarin.Google.Android.DataTransport.Runtime.Dagger.ILazy> (__rm.Handle, JniHandleOwnership.TransferLocalRef);
-			} finally {
-			}
-		}
+		// [Register ("get", "()Lcom/google/android/datatransport/runtime/dagger/Lazy;", "")]
```

Comment:
Looking at the diff for this class:

    #### Type Changed: Xamarin.Google.Android.DataTransport.Runtime.Dagger.Internal.ProviderOfLazy

    Modified base type:

    ```diff
    -Java.Lang.Object
    +System.Object
    ```

    Added constructor:

    ```csharp
    public ProviderOfLazy ();
    ```

    Removed interfaces:

    ```csharp
    Android.Runtime.IJavaObject
    Java.Interop.IJavaPeerable
    JavaX.Inject.IProvider
    System.IDisposable
    ```

    Removed properties:

    ```csharp
    public override Java.Interop.JniPeerMembers JniPeerMembers { get; }
    protected override IntPtr ThresholdClass { get; }
    protected override System.Type ThresholdType { get; }
    ```

    Removed methods:

    ```csharp
    public static JavaX.Inject.IProvider Create (JavaX.Inject.IProvider provider);
    public virtual Java.Lang.Object Get ();
    ```

I suspect the entire class was removed, and we should remove this entire partial class.  Same with `SetFactory`.